tests/func/run/all.yml

```diff
@@ -69,3 +69,11 @@
       tags:
         - test_keypair
   tags: always
+
+- name: Include project tasks
+  include_tasks: project.yml
+  args:
+    apply:
+      tags:
+        - test_project
+  tags: always
```

Comment:
Looked at the test results now, yeah this won't run. We'll need to do the split of Makefile targets and auth files and test playbooks between admin and tenant variants. I think we can mirror what was done in e2e tests and it should work.